reana-ui/src/pages/launchOnReana/LaunchOnReana.js

```diff
@@ -44,10 +44,37 @@ export default function LaunchOnReana() {
     setLoading(true);
     client
       .launchWorkflow(Object.fromEntries(query))
-      .then(({ data: { workflow_id: workflowId } }) => {
-        dispatch(clearNotification);
-        history.push(`/details/${workflowId}`);
-      })
+      .then(
+        ({
+          data: { workflow_id: workflowId, message, validation_warnings },
+        }) => {
+          console.log(validation_warnings);
+          if (validation_warnings) {
+            let warningMessages = [];
+            // Iterate over all keys in validation_warning
+            for (const key in validation_warnings) {
+              if (key === "additionalProperties") {
+                const properties = validation_warnings[key].join(", ");
+                warningMessages.push(
+                  `Unexpected properties found in the REANA specification: ${properties}`
+                );
+              } else {
+                // For other keys, we simply display the key and its value.
+                warningMessages.push(`${key}: ${validation_warnings[key]}`);
+              }
+            }
+
+            message = `The workflow was submitted, but some warnings were issued during validation!`;
```

Comment:
Do you think this should maybe be returned directly by the server in the `message` field?